server/app/analyses/__init__.py

```diff
@@ -0,0 +1,22 @@
+from .parameters import ParametersAnalysis, Parameters, ParametersIndex
+from .time_tracking import TimeTrackingAnalysis, TimeTracking, TimeTrackingIndex
+from .gradients import GradientsAnalysis, Gradients, GradientsIndex
+from .metrics import MetricsAnalysis, Metrics, MetricsIndex
+from .outputs import OutputsAnalysis, Outputs, OutputsIndex
+
+from ..enums import SeriesEnums
+
+
+class AnalysisManager:
+    @staticmethod
+    def get_or_create(run_uuid: str, analysis: str):
+        if analysis == SeriesEnums.GRAD:
```

Comment:
Keep the set of analyses and in a list and match them by class name. analysis.__name__